doc/manual/modules/performance/ref_available-tuned-plug-ins.adoc

```diff
@@ -139,38 +139,92 @@ The powersave level can be specified by using the [option]`radeon_powersave` opt
 
 `bootloader`::
-Adds options to the kernel command line. This plug-in supports only the GRUB 2 boot loader.
+Adds options to the kernel command line. This plug-in supports the GRUB 2 boot loader and Boot Loader Specification (BLS).
 +
 Customized non-standard location of the GRUB 2 configuration file can be specified by the [option]`grub2_cfg_file` option.
 +
 The kernel options are added to the current GRUB configuration and its templates. The system needs to be rebooted for the kernel options to take effect.
 +
-Switching to another profile or manually stopping the `tuned` service removes the additional options. If you shut down or reboot the system, the kernel options persist in the [filename]`grub.cfg` file.
+Switching to another profile or manually stopping the `tuned` service removes the additional options. If you shut down or reboot the system, the kernel options persist in the [filename]`grub.cfg` file and grub environment files.
 +
 The kernel options can be specified by the following syntax:
 +
-[subs=+quotes]
+[subs="+quotes,+macros"]
 ----
-cmdline=[replaceable]_arg1_ [replaceable]_arg2_ ... [replaceable]_argN_
++cmdline+[replaceable]_suffix_=[replaceable]_arg1_ [replaceable]_arg2_ ... [replaceable]_argN_
 ----
 +
+Or with an alternative, but equivalent syntax:
++
+[subs="+quotes,+macros"]
+----
++cmdline+[replaceable]_suffix_=+[replaceable]_arg1_ [replaceable]_arg2_ ... [replaceable]_argN_
+----
++
+Where [replaceable]_suffix_ can be arbitrary (even empty) alphanumeric string which should be unique across all loaded profiles. Good practice is to use the profile name as the [replaceable]_suffix_ (e.g. [option]`cmdline_my_profile`). If there are defined multiple [option]`cmdline` options with the same suffix (e.g. in the parent profiles), then during the profile load/merge the value which was assigned the last time will win (this is the same as with any other plugin options). Finaly, the kernel command line is constructed by concatenation of the all resulting [option]`cmdline` options.
++
+It's also possible to remove kernel options by the following syntax:
++
+[subs="+quotes,+macros"]
+----
++cmdline+[replaceable]_suffix_=-[replaceable]_arg1_ [replaceable]_arg2_ ... [replaceable]_argN_
+----
++
+Such kernel options will be substracted (instead of concatenation) during the final kernel command line construction.
 --
 .Modifying the kernel command line
 ====
 For example, to add the [option]`quiet` kernel option to a *Tuned* profile, include the following lines in the [filename]`tuned.conf` file:
 
 ----
 [bootloader]
-cmdline=quiet
+cmdline_my_profile=+quiet
+----
+
+The following is an example of a custom profile `my_profile` that adds the [option]`isolcpus=2` option to the kernel command line:
+
+----
+[bootloader]
+cmdline_my_profile=isolcpus=2
+----
+
+The following is an example of a custom profile `my_profile` that removes the [option]`rhgb quiet` options from the kernel command line (if previously added by *Tuned*):
```

Comment:
Nit:
s/The following is an example/An example/

Reply:
Fixed.